client/src/planwise/client/scenarios/changeset.cljs

```diff
@@ -81,11 +81,13 @@
         suggestions)])
 
 (defn- changeset-table-row
-  [props {:keys [name change] :as provider}]
+  [{:keys [source-demand] :as props}
+   {:keys [name change satisfied-demand reachable-demand] :as provider}]
```

Comment:
reachable-demand is used so it could be removed here

Reply:
Removed `reachable-demand` and `change` as well since they were not used. Thanks!